src/output.jl

```diff
@@ -26,10 +28,13 @@ end
 plots the species abundance in the landscape for a given timestep t
 """
 function image_abundances(SD::Simulation_Data, t::Int)
-    abundance = SD.species[1].output.abundances[:, :, t]
-    return heatmap(
-        abundance; title="Species Abundance at Timestep $t", c=:YlGnBu, yflip=true
+    abundance = SD.species[1].abundances[:, :, t]
```

Comment:
Needs to be `species[1].output.abundances`

Reply:
done!